### PR TITLE
Add Capistrano completion

### PR DESCRIPTION
**Pull request by MasterRoot24**

### CODE REVIEW

File: cap-completion.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CapCompletion < Formula
+  desc "Bash completion for Capistrano"
+  homepage "https://github.com/bashaus/capistrano-autocomplete"
+  url "https://github.com/bashaus/capistrano-autocomplete/archive/a3a6d3d479a6922a629caf95432c354054567713.tar.gz"
```
Comment (dunn) [inline cap-completion.rb]: We'll need a tagged release.  Could you ask the developers to provide one? Thanks!

**Comment by MasterRoot24** [comment]
@dunn Really? How come [this](https://github.com/Homebrew/homebrew-completions/blob/master/rake-completion.rb) doesn't? It's just pulling from a Gist.

**Comment by dunn** [comment]
@bfontaine do you allow pinning formulae to commits rather than tags in this repo?

File: cap-completion.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  version "1.0.0"
+  sha256 "09fbb136979c01c45b0e6a35152d3152174ed43c07ce08832586680ebbcc418e"
+
+  def install
+    bash_completion.install "cap" => "cap"
```
Comment (bfontaine) [inline cap-completion.rb]: You can use `bash_completion.install "cap"` if both the source and the target names are the same.

**Comment by bfontaine** [comment]
@dunn Yeah; a lot of formulae in this tap are just pulling scripts from gists. Since this one is a GitHub repo and upstream seems reactive we can ask them for a stable release.

**Comment by MasterRoot24** [comment]
I've pushed the suggested change and requested a release from upstream: https://github.com/bashaus/capistrano-autocomplete/issues/4

**Comment by bfontaine** [comment]
Thanks @MasterRoot24!

**Comment by MasterRoot24** [comment]
FYI: Upstream has provided a release and I will update the branch for this PR when I get chance this evening.

**Comment by MasterRoot24** [comment]
Ok, I've updated the formula to install v1.0.0 of @bashaus Cap completion script, as requested.

This should be good to go now!

**Comment by bfontaine** [comment]
```
      ==> Fetching cap-completion from homebrew/completions
==> Downloading https://github.com/bashaus/capistrano-autocomplete/archive/v1.0.0.tar.gz
==> Downloading from https://codeload.github.com/bashaus/capistrano-autocomplete/tar.gz/v1.0.0
Downloaded to: /Library/Caches/Homebrew/cap-completion-1.0.0.tar.gz
SHA256: 66a94420be44d82ff18f366778e05decde3f16ad05d35fd8ec7b51860b102c0c
==> Retrying download
==> Downloading https://github.com/bashaus/capistrano-autocomplete/archive/v1.0.0.tar.gz
==> Downloading from https://codeload.github.com/bashaus/capistrano-autocomplete/tar.gz/v1.0.0
Warning: Formula reports different SHA256: 09fbb136979c01c45b0e6a35152d3152174ed43c07ce08832586680ebbcc418e
Downloaded to: /Library/Caches/Homebrew/cap-completion-1.0.0.tar.gz
SHA256: 66a94420be44d82ff18f366778e05decde3f16ad05d35fd8ec7b51860b102c0c
```

It seems like you need to update the `sha256`.

**Comment by MasterRoot24** [comment]
@bfontaine Sorry about that. I checked the SHA before I pushed, but it seems that the local cache was confusing me and giving me the old SHA.

Updated the SHA in the formula and pushed.

**Comment by MasterRoot24** [comment]
`brew audit --strict cap-completion.rb` doesn't report any errors for me locally, but the @BrewTestBot has reported the build failed. Why is that?

The [build log](http://bot.brew.sh/job/Homebrew%20Completions%20Pull%20Requests/133/#showFailuresLink) shows:
```
homebrew/completions/cap-completion:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

That seems a bit harsh.

**Comment by bfontaine** [comment]
@MasterRoot24 you can get this locally by using `--online`:
```sh
brew audit --strict --online cap-completion.rb
```

This requirement is for the core repository; we’re not that selective in that one so if it’s the only audit failure this pull-request is good to merge.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @MasterRoot24; we appreciate it! 🎉

**Pull request closed without merging**